scripts/update-frontend-canister.sh

```diff
@@ -1,6 +1,20 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+help()
+{
+   # display help
+   echo "Build frontend canister wasm. Copies the wasm build artifact"
+   echo "and corespontind candid file (src/canisters/frontend/ic-certified-assets/assets.did)"
+   echo "to src/distributed/assetstorage.did."
+   echo
+   echo "Options:"
+   echo "  -d, --development-build    build canister using cargo"
+   echo "  -r, --release-build        build canister using linux/amd64 docker image"
```

Comment:
Please update CONTRIBUTING.md and .github/workflows/build-frontend-canister.yml with the release build option